### ISSUE TITLE
Unable to find IP address

### ISSUE THREAD

**Issue by bayek0fsiwa** [subins2000/WebDrop#8]
Showing **Could not find your IP address retry** unable to send files over the internet.

**Comment by subins2000** [comment]
Are you able to access this website in your device : https://ipv4.icanhazip.com/ and https://api.ipify.org/ ?

[public-ip](https://github.com/sindresorhus/public-ip) is used to retrieve IP address, if the sites are blocked or unreachable, this error happens.

**Comment by bayek0fsiwa** [comment]
No, Both sites are blocked, browser showing server IP address could not be found.

**Comment by subins2000** [comment]
Wow, why are they blocked though ? Is this a country thing or an ISP problem ?

Can you also check if this is accessible : https://www.cloudflare.com/cdn-cgi/trace ?

Looks like there should be more ways to find IP address. Didn't anticipate those sites are blocked. Related: https://github.com/sindresorhus/public-ip/issues/45

**Comment by bayek0fsiwa** [comment]
Yes it is accessible: https://www.cloudflare.com/cdn-cgi/trace
Here's The Output:
 👇🏼
fl=35f580
h=www.cloudflare.com
ip=2405:204:a51a:df07:8cbf:6937:fca2:d98
ts=1602243031.604
visit_scheme=https
uag=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/85.0.4183.121 Safari/537.36 Edg/85.0.564.70
colo=SIN
http=http/2
loc=IN
tls=TLSv1.3
sni=plaintext
warp=off

I think it's a DNS or ISP problem

**Comment by subins2000** [comment]
Ah, this is probably because you have an IPv6 address. Both the IP APIs mentioned above uses IPv4. Just one more test, what's the output of https://ip-api.io/api/json ?

**Comment by subins2000** [comment]
@bayek0fsiwa The latest change probably have solved the issue. Can you check please ?

**Comment by bayek0fsiwa** [comment]
Still showing **Could not find your IP address**  and this is not working https://ip-api.io/api/json either 
but I found the solution actually I was using a custom DNS server from Quad9 so after switching to default DNS it's working fine now.

**Comment by subins2000** [comment]
WebDrop app is a progressive web app, so you'll have to reload again to get the new app content. This looks more like an IPv6 issue rather than a block. I've specifically added IPv6 address retrieval in the latest commit.

Once this is merged, cloudflare API can be used too (which gives good result according to the test you did): https://github.com/sindresorhus/public-ip/pull/46

**Comment by AnshumanFauzdar** [comment]
Hey @subins2000 thanks for this wonderful project.
I am facing similar problem but I am using nextDNS so maybe cloudflare fetching can solve this problem?

**Comment by subins2000** [comment]
Hi @AnshumanFauzdar, thank you :)

I've been waiting for my [PR](https://github.com/sindresorhus/public-ip/pull/46) to be merged to public-ip. It's almost a month now, so temporarily I've updated WebDrop's `public-ip` dependency to my fork now. Hope this solves the problem. Can you please see if this change has solved the problem for you ?

**Comment by AnshumanFauzdar** [comment]
Hey @subins2000 I tried but results are same.

**Comment by subins2000** [comment]
@AnshumanFauzdar can you hop over to the Networks tab in browser developer tools and send a screenshot, I'm curious about whether requests to the IP service is going alright.

Also, since WebDrop is a PWA, the app might be cached. You may want to disable the cache in Network tab and reload or wait for the service worker to update the cache (after which you'll see a reload button on bottom right)

**Comment by AnshumanFauzdar** [comment]
@subins2000 there is no error in PC, I have configured next DNS in my android phone.
